### PR TITLE
Background updates fixes 01

### PR DESCRIPTION
**Pull request by florianpreknya**
Small bunch of fixes related do background updates.

### CODE REVIEW

File: nightguard WatchKit Extension/AppMessageService.swift [hunk 1]
```diff
@@ -35,6 +35,21 @@ class AppMessageService : NSObject, WCSessionDelegate {
         }
     }
     
+    func keepAwakePhoneApp() {
+        
+        // send a dummy message to keep the phone app awake (start the app if is not started)
```
Comment (dhermanns) [inline nightguard WatchKit Extension/AppMessageService.swift]: This still feels like a hack, right? It's really a shame how hard it is to keep an app running. Even with background updates running. If I where Apple, I would suggest a special App-Type "Medical" with more rights to consume battery :-/ . But it seems like the running app industry has a better lobby to get such extra rights...

Comment (florianpreknya) [reply nightguard WatchKit Extension/AppMessageService.swift]: Yeah, you're right! It's a hack, but I noticed that the phone app is alive all the time after doing it.
Very true about Apple policies. It seems that if you have a name, like Dexcom, you can obtain all the 288 updates per day. Open source is not loved, instead.. I don't know how easy is to obtain Apple approval if you add HealthKit capabilities to your app. By the way, is there any chance for implementing the background alarms using HealthKit? Or you should start running first, and only after you'll get the haptic feedback working? :)))

**Comment by dhermanns** [comment]
Hi Florian,

how could you implement the alarms using healthkit?

cu,
Dirk

**Comment by florianpreknya** [comment]
https://github.com/mathiasnagler/watchOS-Background-Haptics

**Comment by dhermanns** [comment]
That sounds very interesting: Would be another major improvement.
I don't think that it would be a big issue to get the health kit
approvement.
Would you expect any obstacles from apple?

**Pull request merged**